.reuse/dep5

```diff
@@ -82,3 +82,7 @@ License: Apache-2.0
 Files: internal/compiler/widgets/fluent-base/_*.svg
 Copyright: Copyright © Microsoft Corporation <https://github.com/microsoft/fluentui-system-icons/blob/main/LICENSE>
 License: MIT
+
+Files: internal/compiler/widgets/material-base/_*.svg
+Copyright: Material Icons <https://github.com/material-icons/material-icons/blob/master/LICENSE>
```

Comment:
small details: but i wonder if it would not be better to add these kind of copyright in the svg themselves.
(since these images are uploaded in the crate package for the compiler where .reuse is not available)
But it doesn't matter much anyway.